runtime-sdk/src/dispatcher.rs

```diff
@@ -50,13 +51,28 @@ pub enum Error {
 pub struct DispatchResult {
     pub result: types::transaction::CallResult,
     pub tags: Tags,
+    // Transaction priority.
```

Comment:
Could change these to doc comments (e.g. `///`) and add them to other fields as well.